test-harness/src/test/groovy/com/netflix/conductor/test/integration/ForkJoinSpec.groovy

```diff
@@ -331,13 +331,13 @@ class ForkJoinSpec extends AbstractSpecification {
 
         and: "the workflow is in the failed state"
         with(workflowExecutionService.getExecutionStatus(workflowInstanceId, true)) {
-            status == Workflow.WorkflowStatus.FAILED
```

Comment:
is this change expected?

Reply:
Yes because for the JOIN is marked as async so workflow remains in progress until decide is called.